src/config/dynamic-theme-fixes.config

```diff
@@ -1127,6 +1127,12 @@ CSS
 discord.com
 
 CSS
+path.wavePath-haxJK1 {
+    fill: #1D2021 !important;
+}
+.buttonBrand-33cvhX {
```

Comment:
Where can I find this?

Reply:
![Screenshot_20200816-231212.jpg](https://user-images.githubusercontent.com/32613756/90343998-fe15d480-e015-11ea-9a65-da508ce26c3b.jpg) This "waves"

Reply:
That's the wave that is been fixed by #3422

Reply:
but... my pr is "26 days ago", so why you didn't merge ?

Reply:
Because I probably forgot.